client/components/social-buttons/google.js

```diff
@@ -45,6 +47,10 @@ class GoogleLoginButton extends Component {
 			return this.initialized;
 		}
 
+		this.setState( { error: '' } );
```

Comment:
Yep, `initialize()` can be called multiple times - `this.initialized` won't be set until the `gapi` instance is properly initialised.